ccdproc/__init__.py

```diff
@@ -11,6 +11,15 @@
 from ._astropy_init import *
 # ----------------------------------------------------------------------------
 
+# set up the version
```

Comment:
An alternative is to put this into `version.py` and import. I don't have strong feelings either way.

Reply:
Sounds like for the method below to work we may need to add this to `pyproject.toml:

```
[tools.setuptools_scm]
write_to = ccdproc/version.py
```

Reply:
This won't work yet with the latest stable version of setuptools_scm. A better approach for now is to do this in the ``setup.py`` as done here:

https://github.com/astropy/astropy/blob/baa1914dc700b11ab5d8c32e8c64f0e4f8d9c8a0/setup.py#L62

This has the advantage that e.g. ``python setup.py --version`` will work nicely.

Reply:
And I'm going to update the migration guide to recommend the setup.py approach for now. Then you can just update ``_astropy_init.py`` to import ``version`` from ``.version`` as ``__version__``.

Reply:
FYI, this freezes the version number in a development install at whatever it was when `pip install -e .` is run. As far as I can tell the only workaround would be to run `get_version` on import of `ccdproc`, which would mean requiring `setuptools_scm` for a development install (which is maybe not the end of the world).

Reply:
Note that `python setup.py --version` always gives the correct version because it calls `get_version` via `setup`. If you do `pip install -e .` and then `python -c "from ccdproc import __version__; print(__version__)"` you get the version that existed when `pip` was run.